tests/AntDesign.Tests/Select/Actions/Select.ClearTests.razor

```diff
@@ -0,0 +1,118 @@
+﻿@using System.Net.Sockets
+@inherits AntDesignTestBase
+@code {
+    record Person(int Id, string Name);
+    List<Person> _persons = new()
+    {
+        new(1, "John"),
+        new(2, "Lucy"),
+        new(3, "Jack"),
+        new(4, "Emily"),
+    };
+
+    [Fact]
+    public void DefaultSelectWithoutAllowClear()
+    {
+        JSInterop.Setup<DomRect>(JSInteropConstants.GetBoundingClientRect, _ => true)
+            .SetResult(new DomRect());
+        var cut = Render<Select<int, Person>>(
+            @<AntDesign.Select DataSource="@_persons"
+                               LabelName="@nameof(Person.Name)"
+                               ValueName="@nameof(Person.Id)"
+                               Value="0">
+            </AntDesign.Select>);
+        Assert.Throws(typeof(ElementNotFoundException), () => cut.Find("span.ant-select-clear"));
```

Comment:
My xunit is throwing here a warning: `Do not use typeof() expression to check the exception type.`.
I would advise to change this and further exception asserts to this line of code:
```csharp
cut.Invoking(c => c.Find("span.ant-select-clear")).Should().Throw<Bunit.ElementNotFoundException>();
```